plugins/services/src/js/utils/MarathonErrorUtil.js

```diff
@@ -121,6 +121,15 @@ const MarathonErrorUtil = {
         });
       }
 
+      if (!Array.isArray(errors) || errors.length === 0) {
```

Comment:
It's by no means a better solutions... I've changed the implementation to reuse the reducer.